core/src/main/scala/chisel3/Annotation.scala

```diff
@@ -20,6 +20,14 @@ trait ChiselAnnotation {
   def toFirrtl: Annotation
 }
 
+/** Enhanced interface for Annotations in Chisel
+  *
+  *  Defines a conversion to corresponding FIRRTL Annotation(s)
+  */
+trait ChiselToFirrtlAnnotations {
+  def toFirrtlAnnotations: Seq[Annotation]
```

Comment:
```suggestion
trait ChiselMultiAnnotation {
  def toFirrtl: Seq[Annotation]
```

Meganit but just making the naming be more in line with the existing `ChiselAnnotation`

Reply:
Changed!